packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -37,33 +34,33 @@ export class LayeredRuntimeResolverCreator {
         const resolverCreateField = getFieldName(modelName, GraphbackOperationType.CREATE);
         //tslint:disable-next-line: no-any
         resolvers.Mutation[resolverCreateField] = (parent: any, args: any, context: any) => {
-          if (!this.services[modelName]) {
+          if (!context.services[modelName]) {
             throw new Error(`Missing service for ${modelName}`);
           }
 
-          return this.services[modelName].create(args.input, context)
+          return context.services[modelName].create(args.input, context)
```

Comment:
Awesome! This addresed problem with ability to use services in your own resolver if needed.

Reply:
Docs needed